src/pages/en/guides/images.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Images
+description: Learn how to use images in Astro.
+i18nReady: true
+setup: |
+  import Since from '../../../components/Since.astro';
+---
+Astro provides several ways for you to use images inside of your projects.
+
+### `.astro` files
+
+Astro uses standard HTML `<img>` or JSX `<img />` elements to display images within your `.astro` files. 
+
+```astro
+---
+// src/pages/index.astro
+import rocket from './images/rocket.svg';
+---
+<img src="https://astro.build/assets/press/full-logo-light.png" alt="Astro">
+<img src="/stars.png" alt="A starry night sky."> <!-- stored in public -->
+<img src="./images/astronaut.jpg" alt="An astronaut."> <!-- stored in src -->
```

Comment:
```suggestion
<img src="./images/astronaut.jpg" alt="An astronaut."> <!-- relative path from file -->
```

Reply:
I changed these to reflect the actual location of the file. Tell me what you think!